test/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ set(CORE_TEST_SOURCES
     test_column_float.cpp
     test_column_string.cpp
     test_column_timestamp.cpp
+	test_db.cpp
```

Comment:
indentation should be in line with the surrounding style

Reply:
Interesting. It's a tab instead of spaces which looks the same on my side. Fixed with 6009d8673.